pandas/io/html.py

```diff
@@ -1058,6 +1106,11 @@ def read_html(
     displayed_only : bool, default True
         Whether elements with "display: none" should be parsed.
 
+    extract_hrefs : all/header/body/footer or None, default None
+        Table elements in the specified section(s) with <a> tags will have their
+        href extracted. Note that specifying "header" will result in a
```

Comment:
not 100% sure i love the argument name., cc @attack68

Reply:
I feel like `href` is more explicit than `link`, so I'm not sure if `extract_links` is better, but if you prefer it, I'll go ahead with it. Thoughts?

Reply:
`extract_urls` perhaps?

Reply:
The issue is that not all hrefs are URLs or even links (e.g. in the case where they are JS or a `mailto`, etc.).